# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/bpftool/internal-hdrs-install.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- src/Makefile.vanilla	2026-02-12 19:46:40.267000000 +0100
+++ src/Makefile	2026-02-12 19:46:53.481822120 +0100
@@ -40,7 +40,7 @@
 
 $(LIBBPF_INTERNAL_HDRS): $(LIBBPF_HDRS_DIR)/%.h: $(BPF_DIR)/%.h | $(LIBBPF_HDRS_DIR)
 	$(call QUIET_INSTALL, $@)
-	$(Q)install -m 644 -t $(LIBBPF_HDRS_DIR) $<
+	$(Q)/usr/bin/install -m 644 -t $(LIBBPF_HDRS_DIR) $<
 
 $(LIBBPF_BOOTSTRAP): $(wildcard $(BPF_DIR)/*.[ch] $(BPF_DIR)/Makefile) | $(LIBBPF_BOOTSTRAP_OUTPUT)
 	$(Q)$(MAKE) -C $(BPF_DIR) OBJDIR=$(patsubst %/,%,$(LIBBPF_BOOTSTRAP_OUTPUT)) \
